interface/controller/item_controller.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"academy_bootcamp/application/usecase"
+	"academy_bootcamp/domain/model"
+)
+
+type itemController struct {
+	Usecase usecase.ItemService
+}
+
+type ItemController interface {
+	GetItemById(c Context) error
+	// CreateItem(c Context) error
+}
+
+func NewItemController(us usecase.ItemService) ItemController {
+	return &itemController{us}
+}
+
+func (uc *itemController) GetItemById(c Context) error {
+	var item *model.Item
+	key, err := strconv.ParseUint(c.Param("id"), 10, 16)
+	if err != nil {
+		return errors.New("QueryParam invalido")
+	}
+
+	item, err2 := uc.Usecase.GetByKey(key)
```

Comment:
If you make a request with an inexisting id, it is returning a `null` and a `200 OK` response. Please fix this.